data/human/human missions.txt

```diff
@@ -7453,3 +7453,405 @@ mission "Adelita 5"
+			`	He pauses, letting the weight of his words sink in. "Getting in on this is a once-in-a-lifetime chance. We're talking about a key hub on a high-traffic trade route, with a captive market of ships desperate for fuel. The returns will be solid... and dependable. Opportunities like this don't just come around, and when they do, they go fast."`
+			choice
+				`	"Okay."`
+			branch upcharge
+				"syndicate business: mean to Howard" > 0
+			`	"Excellent. The price for this investment is ten million credits for a solid 2700 credit per day return. Don't worry about pulling out your wallet just yet--I've pulled a few strings with our friends at the bank. They're willing to structure a loan for you through our Finance Division, no upfront payment needed. You'll just need to sign a few documents to get things moving."`
+			`	He slides a document over to you, with a sleek pen ready for your signature. "Just sign this first document for the loan and then we can talk through the rest of documents before signing."`
+			choice
+				`	"Sounds good."`
+					goto nextynice
+				`	"I'm done with these deals, I'm leaving."`
+					decline
+			label upcharge
+			`	Howard's tone carries a hint of impatience. "Alright then. The price for this investment is eleven million credits for a 2700 credit per day return. I've managed to arrange a loan through the bank and our Finance Division, so you won't need to come up with the cash immediately. Just sign these documents to formalize the arrangement."`
+			`	He pushes a document over to you, with a very fancy pen ready for your signature. "Just sign this first document for the loan and then we can talk through the rest of documents before signing."`
+			choice
+				`	"Sounds good."`
+					goto nextymean
+				`	"I'm done with these deals, I'm leaving."`
+					decline
+			label nextynice
+			action
+				debt 10000000
+					interest 0.08
+					term 3650
+			`	He takes the document from you and you see the loan terms on the screen behind him: 10M credits, 10 years, 34% rate.`
+				goto nexty
+			label nextymean
+			action
+				debt 11000000
+					interest 0.10
+					term 3650
+			`	He snatches the document from you with a wicked smile and you see the loan terms on the screen behind him: 11M credits, 10 years, 44% rate.`
+			label nexty
+			`	Howard leans back in his chair. "Well done. You're now part of a very select group, one that's about to see some significant returns. As I mentioned, we're leaning towards a conventional expansion for the refueling station on Crossroads. It's a strategic move and it'll serve the steady stream of ships passing through, locking in profits for years to come of about 2700 credits a day, or around 10% a year."`
+			`	He watches you closely, gauging your reaction. "Of course, this is just the standard approach. There's always room to tweak the plan."`
+			`	Howard pauses, waiting to see if you're interested in taking the conversation further.`
+			choice
```

Comment:
One more thing about this sequence: the questions can repeat. Ex. "What would it take to boost profits even more?" can be chosen even if you've already asked.

Reply:
That is intentional so that the player can return to pick an option they previously viewed. Should I change the phrasing to be more fitting for a repeat selection?

Reply:
That's what I was thinking. They should definitely be able to return.

Reply:
And done! A bit more finicky to implement than I initially thought.